docs/DeveloperGuide.md

```diff
@@ -112,39 +112,39 @@ The `UI` component,
 
 <img src="images/LogicClassDiagram.png" width="800"><br>
 
-**API** : [`Logic.java`](../src/main/java/seedu/address/logic/Logic.java)
+**API** : [`Logic.java`](../src/main/java/tars/logic/Logic.java)
```

Comment:
👍 I like how you guys updated the diagrams with your own classes up till now ~ well done :) But do take note to not only add on but also understand the current underlying structure that is provided.